bin.js

```diff
@@ -611,24 +611,24 @@ async function runTool (args, Tool, version, uiOptions) {
     console.log(`Output file is ${outputs.data}`)
   }
 
-  if (Tool.name === 'ClinicDoctor' && !args['collect-only']) {
-    try {
-      const iss = tool.getIssue()
-      const proc = args['--'][args['--'].length - 1]
-      if (iss !== 'none') {
-        console.log(`Doctor detected a potential ${iss} issue.`)
-      }
-      if (iss === 'event-loop') {
-        console.log('To get started with diagnosing the issue run:')
-        console.log(`clinic flame --autocannon [ / ] -- node ${proc}`)
-      } else if (iss === 'io') {
-        console.log('To get started with diagnosing the issue run:')
-        console.log(`clinic bubbleprof --autocannon [ / ] -- node ${proc}`)
-      } else if (iss === 'data') {
-        console.log('Try running the benchmark for a longer time.')
-      }
-    } catch (err) {
-      // The version of Doctor used does not have a getIssue() function
+  if (Tool.name === 'ClinicDoctor' && !args['collect-only'] && !args['visualize-only']) {
+    const iss = tool.issue
+
+    const argsArr = args['--']
```

Comment:
The `.shift()` call below will mutate the original `args['--']` array, which maaaay not be a problem in this case but would definitely be unexpected if some other place in the code needs it in the future.

You can create a copy using the `.slice()` method, but even better, here you can do:

```js
args['--'].slice(1)
```
to get a new array that has all the elements from the `args['--']` array, _skipping_ the 1st element.

Reply:
Thanks! That's sorted now